apps/projects/management/commands/util.py

```diff
@@ -0,0 +1,101 @@
+import pytz
+import requests
+
+from django.core.management.base import CommandError
+from django.utils import timezone
+
+from adhocracy4.modules.models import Module
+from adhocracy4.phases.models import Phase
+from adhocracy4.projects.models import Project
+
+
+def a3_login(url, headers, username, password):
+    login_url = url + 'login_username'
+    res = requests.post(
+        login_url,
+        json={'name': username, 'password': password}
+    )
+    if res.status_code != requests.codes.ok:
+        raise CommandError('API user authentication failed.')
+    headers['X-User-Token'] = res.json()['user_token']
+
+
+def a3_get_elements(url, headers, resource_type, elements):
+    query_url = '{}?content_type={}&depth=all&elements={}'.format(
+        url, resource_type, elements
+    )
+    res = requests.get(query_url, headers=headers)
+    if res.status_code != requests.codes.ok:
+        raise CommandError('Request failed for URL: {}'.format(query_url))
+    data = res.json()
+    paths = data['data']['adhocracy_core.sheets.pool.IPool']['elements']
+    return paths
+
+
+def a3_get_sheet_field(resource_url, headers, sheet, field):
+    res = requests.get(resource_url, headers=headers)
+    if res.status_code != requests.codes.ok:
+        raise CommandError('Request failed for URL: {}'.format(
+            resource_url)
+        )
+    data = res.json()
+    sheet_field_value = data['data'][sheet][field]
+    return sheet_field_value
```

Comment:
If this gets too slow, a simple optimization could be to define `a3_get_resource()` and decorate it with `functools.lru_cache()`.